### PR TITLE
Batch create mutation

### PR DESCRIPTION
**Pull request by star2000**
can do
```graphql
mutation {
  createItems(data: [{name: "item1"}, {name: "item2"}]) {
    id
  }
}
```
Keep the `createItem` at the same time.

### CODE REVIEW

File: strawberry_django/resolvers.py [hunk 1]
```diff
@@ -48,9 +48,10 @@ def get(self, id):
 
     def create(self, data):
         model = self.get_model()
-        instance = model(**data)
-        instance.save()
-        return instance
+        instances = [model(**i) for i in data]
+        for instance in instances:
+            instance.save()
```
Comment (la4de) [inline strawberry_django/resolvers.py:53]: I'd prefer bulk creation. It's more efficient. What do you think?

https://docs.djangoproject.com/en/3.1/ref/models/querysets/#bulk-create

**Review by la4de — CHANGES_REQUESTED** [review]
First of all I want to thank you for your contribution.

I'd prefer the name `createItems`.

Please add unit test for batch creation. You can run unit tests by executing following commands under the repo
```
poetry install
poetry run pytest
```
Prerequisite is that you have `poetry` installed and available.

File: strawberry_django/resolvers.py (hunk 1 above)
Comment (star2000) [reply strawberry_django/resolvers.py:53]: Tried, this method can't return the ID field

**Comment by star2000** [comment]
> First of all I want to thank you for your contribution.
> 
> I'd prefer the name `createItems`.
> 
> Please add unit test for batch creation. You can run unit tests by executing following commands under the repo
> 
> ```
> poetry install
> poetry run pytest
> ```
> 
> Prerequisite is that you have `poetry` installed and available.

complete

**Comment by star2000** [comment]
@la4de

**Comment by la4de** [comment]
Looks good to me. Thank you very much!

**Pull request merged**